javascript-practice/src/index.html

```diff
@@ -10,8 +10,53 @@
 
 <body>
 	<!-- START HEADER SECTION -->
-	<header></header>
-	<!-- END HEADER SECTION -->
+	<header class="header">
+		<div class="top-nav">
+			<span class="header-search">
+				<img class="search-icon" src="assets/icons/search.svg" alt="icon search" />
+				<img class="close-icon" src="assets/icons/close.svg" alt="icon close" />
```

Comment:
Why you have 2 images in a span? So confused, could you help to check it again? @mthanh2209

Reply:
Thanks. I fixed it.